### PR TITLE
Schema update for task IDs and tips.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: assets/questions/findFirstNonRepeatingCharacter.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ globalData.questions['findFirstNonRepeatingCharacter'] = { // eslint-disable-lin
 `
   },
   tasks: [{
+    id: 'findFirstNonrepeatingCharacter',
```
Comment (eyurko) [inline assets/questions/findFirstNonRepeatingCharacter.js]: findFirstNonRepeatingCharacter, for consistency.

Comment (seanlip) [reply assets/questions/findFirstNonRepeatingCharacter.js]: Done.

**Comment by seanlip** [comment]
Hi @eyurko, can this go in?

**Comment by eyurko** [comment]
Sorry about that -- approved!

**Comment by eyurko** [comment]
Yo -- this doesn't push taskIndex as a string -- it leaves it as an int currentTaskIndex (see LearnerViewDirective). Is that intentional, or did you mean to do that in this CL?

**Comment by seanlip** [comment]
Ah, no, this PR doesn't convert taskIndex to taskId, or change anything related to events. It just adds a taskId field to the Task schema so that it can be used in the first place.

**Pull request merged**